packages/stops-overlay/src/default-stop-marker.js

```diff
@@ -5,7 +5,7 @@ import PropTypes from "prop-types";
 import React, { Component } from "react";
```

Comment:
On line 1 of this file the import hasn't been update to match the new name of the style export from base-map, so it should be "Styled" instead of "style"